google-cloud-tools-plugin/google-account/.gitignore

```diff
@@ -1,2 +1,3 @@
 /out
+/build
```

Comment:
I dont' think we need this file now since the module now lives in the cloud tools plugin